spec/controllers/api/v1/blood_pressures_controller_spec.rb

```diff
@@ -2,17 +2,18 @@
 
 RSpec.describe Api::V1::BloodPressuresController, type: :controller do
   let(:request_user) { FactoryBot.create(:user) }
+
   before :each do
-    request.env['X_USER_ID'] = request_user.id
+    request.env['HTTP_X_USER_ID'] = request_user.id
```

Comment:
Was this not working before?

Reply:
i think this header was being overridden in the actual specs